crates/sound/README.md

```diff
@@ -30,7 +34,7 @@ generated with help2man target/debug/vhost-user-sound |mandoc
 Launch the backend on the host machine:
 
 ```shell
-host# vhost-user-sound --socket /tmp/snd.sock --backend null
+host# vhost-user-sound --socket /tmp/snd.sock null
```

Comment:
I like the idea of using ValueEnum, but can we still have the `--backend` option?
I always found difficulty in positional arguments.

Reply:
Sure, will do :)